### ISSUE TITLE
Grafana 6.6.2 -> 7.10 - Exporting table with value mappings now exports the key, not the mapped value

### ISSUE THREAD

**Issue by dannyman00123** [grafana/grafana#26479]
**What happened**:
Tried to export table visualisation. The query returns arbitrary integers which I map to Text using Grafana. Usually when exporting, the mapped values are exported. However, since upgrading to 7.1.0, it now exports the integers instead of the mapped values

**What you expected to happen**:
Exported data would be of the value mappings.

**How to reproduce it (as minimally and precisely as possible)**:
1. Create Table visualisation, return some kind of result with random integers in it. I use Clickhouse so used: select * from system.numbers limit 5.
2. this query returns me 5 rows of 0,1,2,3,4
3. Set up a value mapping of 0 -> "test".
4. Try to export Data, and you'll get 0,1,2,3,4

**Anything else we need to know?**:

Occurred when upgrading from 6.6.2 to 7.1.0

**Environment**:
- Grafana version: 7.10
- Data source type & version: N/A
- OS Grafana is installed on: Ubuntu 18.04
- User OS & Browser: Windows 10 Chrome, Firefox
- Grafana plugins: Standard Table
- Others:

**Comment by ivanahuckova** [comment]
Yeah, I was able to reproduce this as well on grafana.play. There is a switch in **Data tab** to display with options defined in the Field and Override tabs, but this switch doesn't have any effect on CSV export. 

![image](https://user-images.githubusercontent.com/30407135/88168956-7d80d580-cc1b-11ea-856f-7f3ace9bf4ec.png)

**Comment by torkelo** [comment]
@ryantxu Need to think about how support display processors in CSV export. Currently, we do not use one (except a hacky workaround for the first time field).

So the CSV export does not support value mappings as that is in the display processor. But if the CSV export started using the display processor we would run into problems of exporting the raw unparsed values (with full decimals), and the problem of parsing all strings as numbers (here could set unit to string as a workaround).

Related issue: https://github.com/grafana/grafana/issues/25130
